tests/migration/Core/V6_6/Migration1735807464AddCustomFieldStoreApiAwareTest.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_6;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Migration\V6_6\Migration1735807464AddCustomFieldStoreApiAware;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+#[CoversClass(Migration1735807464AddCustomFieldStoreApiAware::class)]
+class Migration1735807464AddCustomFieldStoreApiAwareTest extends TestCase
+{
+    use KernelTestBehaviour;
+
+    private Connection $connection;
+
+    protected function setUp(): void
+    {
+        $this->connection = static::getContainer()->get(Connection::class);
+    }
+
+    public function testCustomFieldHasStoreApiAwareColumn(): void
+    {
+        $this->rollback();
+        $this->executeMigration();
```

Comment:
this should be executed twice, to make sure, the migration could be executed multiple times, but the column is only added once without failure